src/sections/PrevImage.tsx

```diff
@@ -0,0 +1,76 @@
+
+
```

Comment:
Ideally, this component should be called `PrevEventImages`

Reply:
Renamed